### PR TITLE
Update quick tour guide

### LINKED ISSUE #884: [Quick Tour] Improvements In "DEPLOYING ON DOCKER" section
**Description:**
In the "DEPLOYING ON DOCKER" section, "This needs to be added under the `endpoint http:Listener` .." text line is wrong. It should be "before the `endpoint ` ..."

image

And we have to add a Tip after the sample code: 
"On windows PC's make sure the docker runs with linux containers and in the general settings, "Expose daemon on tcp://localhost:2375 without TLS" should be ticked (enabled). For more details check [https://github.com/ballerinax/docker/blob/master/samples/README.md](https://github.com/ballerinax/docker/blob/master/samples/README.md)".

**Affected Product Version:**
v0.9.016

**OS, DB, other environment details and versions:**    
All

**Steps to reproduce:**
1. Navigate to: https://ballerina.io/learn/quick-tour/
2. Scroll down to "DEPLOYING ON DOCKER" section

### PR DESCRIPTION
**Pull request by pubudu91**

### CODE REVIEW

File: website/two-column-pages/docs/learn/quick-tour.md [hunk 1]
```diff
@@ -229,6 +229,43 @@ twitter:Status st = check twitter->tweet(status);
...
+};
+
+endpoint http:Listener tweetEP {
+    port: 9090
+};
+
+@http:ServiceConfig {
+    basePath: "/"
+}
+service<http:Service> hello bind listener {
```
Comment (Anoukh) [inline website/two-column-pages/docs/learn/quick-tour.md]: Shouldn't this be `bind tweetEP`

File: website/two-column-pages/docs/learn/quick-tour.md [hunk 2]
```diff
@@ -379,66 +424,6 @@ For more information on the `ballerina push` command run the following.
...
-        http:Response response = new;
-        response.setTextPayload("ID:" + <string>st.id + "\n");
-        response.setTextPayload("Hello Ballerina!\n");
-        _ = caller -> respond(response);
-    }
-}
-
-```
-
 ## Run the Composer
```
Comment (samgnaniah) [inline website/two-column-pages/docs/learn/quick-tour.md]: Shall we add how to start the composer via the new electron app?

Comment (pubudu91) [reply website/two-column-pages/docs/learn/quick-tour.md]: Composer team said they'll look into those details.

File: website/two-column-pages/docs/learn/quick-tour.md (hunk 1 above)
Comment (pubudu91) [reply website/two-column-pages/docs/learn/quick-tour.md]: Fixed with commit 35c1543

**Pull request merged**